catalog/security/catalog-security-filter/src/main/java/ddf/catalog/security/filter/plugin/FilterPlugin.java

```diff
@@ -66,13 +87,21 @@ public CreateRequest processPreCreate(CreateRequest input) throws StopProcessing
                 CollectionPermission.CREATE_ACTION);
         List<Metacard> metacards = input.getMetacards();
         Subject subject = getSubject(input);
+        Subject systemSubject = getSystemSubject();
+        List<String> notPermittedIds = new ArrayList<>();
         for (Metacard metacard : metacards) {
             Attribute attr = metacard.getAttribute(Metacard.SECURITY);
             if (!checkPermissions(attr, securityPermission, subject,
-                    CollectionPermission.CREATE_ACTION)) {
-                throw new StopProcessingException("Metacard creation not permitted.");
+                    CollectionPermission.CREATE_ACTION) || !checkPermissions(attr,
+                    securityPermission, systemSubject, CollectionPermission.CREATE_ACTION)) {
```

Comment:
Why do we need to see if the systemSubject has permission to do the operation? Do we pass off the action to the system user at some point?